src/app/modules/form/public-form/public-form.service.ts

```diff
@@ -42,6 +44,7 @@ export const insertFormFeedback = ({
   return ResultAsync.fromPromise(
     FormFeedbackModel.create({
       formId,
+      formSubmissionId,
```

Comment:
what's the behaviour if this is `undefined` (i.e. if the old endpoint is used)? does the property not get created in the db at all, or does it get created with an `undefined` value?

Reply:
The property does not get created in the db at all. This doesn't cause an issue with adding feedbacks and also getting feedbacks using both the new and old endpoints